filechanges/boot/config/settings.ini

```diff
@@ -331,13 +330,46 @@ playlists_dir = /var/lib/mopidy/playlists
 data_dir = /var/lib/mopidy/localdata
 #This sets the method of indexing local files
 #You can set it to json or whoosh. Whoosh might be better with large collections, JSON is a bit more mature.
-# TODO: You cannot add settings for [local-sqlite]
-# because it breaks the startup script (won't read dashes in section names
 library = sqlite
 #library = whoosh
 #library = json
 
 scan_timeout = 1000
-scan_flush_threshold = 1000
+scan_flush_threshold = 100
 excluded_file_extensions = .html, .jpeg, .jpg, .log, .nfo, .png, .txt, .mkv, .avi, .divx, .qt, .wmv, .htm, .zip, .rar, .gz, .pdf, .exe, .ini, .mid, .db, .m3u, .sfv, .midi
 
+[local-sqlite]
+enabled = true
+
+# NOTE: Handling of complex parameters (e.g. that contain spaces and do not contain '=') are not

```

Comment:
I've added a note and commented out the lines that cannot currently be parsed by read_ini.sh